atlasreader/data/README.md

```diff
@@ -0,0 +1,337 @@
+- Simon B Eickhoff, Klaas E Stephan, Hartmut Mohlberg, Christian Grefkes, Gereon R Fink, Katrin Amunts, Karl Zilles (2005). A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. Neuroimage, 25, 1325–1335.
+- Kegang Hua, Jiangyang Zhang, Setsu Wakana, Hangyi Jiang, Xin Li, Daniel S Reich, Peter A Calabresi, James J Pekar, Peter C M van Zijl, Susumu Mori (2008). Tract probability maps in stereotaxic spaces: analyses of white matter anatomy and tract-specific quantification. Neuroimage, 39, 336–347.
+- J Mazziotta, A Toga, A Evans, P Fox, J Lancaster, K Zilles, R Woods, T Paus, G Simpson, B Pike, C Holmes, L Collins, P Thompson, D MacDonald, M Iacoboni, T Schormann, K Amunts, N Palomero-Gallagher, S Geyer, L Parsons, K Narr, N Kabani, G Le Goualher, D Boomsma, T Cannon, R Kawashima, and B Mazoyer (2001). A probabilistic atlas and reference system for the human brain: International Consortium for Brain Mapping (ICBM). The Royal Society Philosophical Transactions B, 356, 1293–1322.
+- Jack L. Lancaster, Marty G. Woldorff, Lawrence M. Parsons, Mario Liotti, Catarina S. Freitas, Lacy Rainey, Peter V. Kochunov, Dan Nickerson, Shawn A. Mikiten, Peter T. Fox (2000). Automated Talairach atlas labels for functional brain mapping. Human Brain Mapping, 10, 120–131.
+- TEJ Behrens, H. Johansen-Berg, MW Woolrich, SM Smith, CAM Wheeler-Kingshott, PA Boulby, GJ Barker, EL Sillery, K. Sheehan, O. Ciccarelli , AJ Thompson, JM Brady, PM Matthews (2003). Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nature Neuroscience, 6, 750–757.
+- Jörn Diedrichsen, Joshua H Balsters, Jonathan Flavell, Emma Cussans, Narender Ramnani (2009). A probabilistic MR atlas of the human cerebellum. Neuroimage, 46, 39-46.
+
+### License
+
+The FSL templates and atlases are under the License that can be found
+[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Licence).
+
+## ICBM 2009c Nonlinear Asymmetric
+
+A number of unbiased non-linear averages of the MNI152 database have been
+generated that combines the attractions of both high-spatial resolution and
+signal-to-noise while not being subject to the vagaries of any single brain
+(Fonov et al., 2011). The procedure involved multiple iterations of a process
+where, at each iteration, individual native MRIs were non-linearly fitted to
+the average template from the previous iteration, beginning with the MNI152
+linear template. The templates present an unbiased standard magnetic resonance
+imaging template brain volume for normal population. These volumes were created
+using data from ICBM project. For more information see: http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009.
+
+### Creation of template
+
```

Comment:
This section is awesome. It’s so simple to “just make a slightly different version” but I remember spending SO LONG as a grad student trying to figure this out! 🙌🏻